javascript/boilerplate-text/boilerplate-zh.js

```diff
@@ -0,0 +1,136 @@
+// translate the text between quotes
+// text following double slashes (such as this line) provides comments for the translator
+// words beginning with s. or f. are variables - do not disturb these ! (though you can move them)
+// where it would aid clarity, variables that occur in the text are described below
+
+var s = { }
+
+// DO NOT TRANSLATE
```

Comment:
It says "DO NOT TRANSLATE", but I see that `s.currLang` in other languages are also translated. @r12a Do you know the reason?

Reply:
When the translator receives the file, these are already pre-translated.